src/ui/DateRangePicker.js

```diff
@@ -101,14 +101,20 @@ class DateRangePicker extends Component {
       return [startDate, endDate, window];
     }
 
-    if (_.isUndefined(start) && _.isUndefined(end)) {
-      const [startDate, endDate] = ranges[dateFormat];
-      return [startDate, endDate, dateFormat];
+    let startDate = null;
+    let endDate = null;
+
+    if (
+      (_.isUndefined(start) && _.isUndefined(end))
+      || (!moment.utc(start).isValid() || !moment.utc(end).isValid())
+    ) {
+      startDate = moment.utc().startOf('day');
```

Comment:
we should eventually make these moments in the browser's timezone instead of forcing utc. At least in the react-date-picker, the ranges will include the day before or after, relative to the browser's local time.

Reply:
Understood. I'll make them so when migrating to react-date-picker.